libs/features/mas/mas/src/mas.js

```diff
@@ -19,5 +19,5 @@ const config = () => ({
 init(config);
 
 if (features.includes('merch-card')) {
-    import(`${origin}/libs/deps/merch-card-all.js`);
+    import('https://main--mas--adobecom.hlx.live/libs/merch-card-all.js');
```

Comment:
Any chance this could be proxied to our main hosts? Loading this on adobe.com / business.adobe.com will come with some TLS handshake penalties that will impact performance.

Reply:
can you import the following instead using relative paths:
```
deps/mas/merch-card.js
deps/mas/merch-icon.js
deps/mas/merch-datasource.js. (not sure if it is there)
```

Reply:
and this should not be coming from mas, but milo @Axelcureno